android-components/components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/storage/SyncedTabsStorage.kt

```diff
@@ -42,6 +43,19 @@ class SyncedTabsStorage(
 ) : SyncedTabsProvider {
     private var scope: CoroutineScope? = null
 
+    // TODO; Pass this value in from the constructor so that it can be set from the Fenix integration side.
+    private val maxActiveTime: Long = TimeUnit.DAYS.toMillis(14)
```

Comment:
Do we want to specifically handle this as a follow-up, or could we potentially do it as part of this patch?